src/Reusable/FileCard.js

```diff
@@ -42,14 +43,33 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+const API_URL = 'http://localhost:8080/file';
 export default function FileCard(props) {
+  
   const classes = useStyles();
   const [expanded, setExpanded] = React.useState(false);
 
   function handleExpandClick() {
     setExpanded(!expanded);
   }
 
+  function deleteFile(){
+    const url = API_URL + '/deleteFile';
+    const fileData = { id:props.fileId,
+                      fileName:props.fileName,
+                      fileType:props.fileType,
+                      fileSize:props.fileSize,
+                      senderId:props.senderId,
+                      receiverId:props.receiverId,
+                      ownerId:props.ownerId,
+                      fileURI:props.fileURI};
+      axios.post(url,fileData )
+            .then(res => {
+      alert(res.data.message);
+})
+    alert("Do you really want to delete "+props.fileName+"?")
```

Comment:
I Believe The file is deleted regardless of the user input. Even if the user does not confirm. Please check.